From 500c490f669062829c8b308d4d66b9be21de9199 Mon Sep 17 00:00:00 2001
From: V8 Autoroll <v8-ci-autoroll-builder@chops-service-accounts.iam.gserviceaccount.com>
Date: Tue, 10 Sep 2024 10:59:18 -0700
Subject: [PATCH] Version 12.8.374.32

Change-Id: I06945033dfb835e55733eff72085cb00d6542f8c
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/5851062
Bot-Commit: v8-ci-autoroll-builder <v8-ci-autoroll-builder@chops-service-accounts.iam.gserviceaccount.com>
Cr-Commit-Position: refs/branch-heads/12.8@{#64}
Cr-Branched-From: 70cbb397b153166027e34c75adf8e7993858222e-refs/heads/12.8.374@{#1}
Cr-Branched-From: 451b63ed4251c2b21c56144d8428f8be3331539b-refs/heads/main@{#95151}
---

--- a/v8/include/v8-version.h
+++ b/v8/include/v8-version.h
@@ -11,7 +11,7 @@
 #define V8_MAJOR_VERSION 12
 #define V8_MINOR_VERSION 8
 #define V8_BUILD_NUMBER 374
-#define V8_PATCH_LEVEL 31
+#define V8_PATCH_LEVEL 32
 
 // Use 1 for candidates and 0 otherwise.
 // (Boolean macro values are not supported by all preprocessors.)
